Makefile

```
@@ -211,115 +158,100 @@ tidy: clean
 
 # Get Python modules.
 
-$(BIN)/pip: $(ACTIVATE_VENV)
-	. $(ACTIVATE_VENV) && $(SEMPIP) pip install --upgrade pip setuptools
-
 %/bin/activate:
-	$(SEMPIP) $(PYTHON) -m venv $(subst /bin/activate,,$@)
-	. $@ && $(SEMPIP) pip install --upgrade pip setuptools wheel
-
-.PHONY: wheel
-wheel: $(BIN)/wheel
-$(BIN)/wheel: | $(ACTIVATE_VENV)
-	. $(ACTIVATE_VENV) && $(SEMPIP) pip install --force-reinstall $(PIP_OPTIONS_E) wheel
+	$(PYTHON) -m venv $(subst /bin/activate,,$@)
+	. $@ && pip install --upgrade pip setuptools wheel
+	. $@ && pip install click requests pyaml ocrd dotenv
```

```suggestion
	. $@ && pip install click requests pyyaml ocrd dotenv
```

Unless we need some functionality from pyaml, which I never use.